Automattic/wp-calypso
# Payments: Update credit card fields design

#### Changes proposed in this Pull Request

This PR makes a number of changes to cleanup and update the design of the add/change credit card form fields. Some of the changes include:
- Consistent font styles for Stripe and regular fields
- Removed UI jump after stripe fields load
- Cleanup form labels case
- Remove unused popup component (replaced with inline graphic for CVV field)
- Consistent column widths
- Improved form loading animations

**Before**
![image](https://user-images.githubusercontent.com/6981253/99403128-1e288e80-28b8-11eb-8be5-83158baf4e42.png)


**After**
![image](https://user-images.githubusercontent.com/6981253/99403019-018c5680-28b8-11eb-855e-dc5df34f3de3.png)

#### Testing instructions



* Visit site and account level screens for adding/changing a credit card.
* Confirm the form loads correctly and everything looks right.

Please note that If you encounter an issue when you pick Brazil (ebanx), the resulting crash is an existing issue and not related to this PR.
